apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -159,6 +160,9 @@ export default function ScheduleCalendar(props: ScheduleCalendarProps) {
     const calendarStyling = isMobile ? { height: `calc(100% - 55px)` } : { height: `calc(100vh - 104px)` };
     const calendarTimeFormat = isMilitaryTime ? 'HH:mm' : 'h:mm A';
     const calendarGutterTimeFormat = isMilitaryTime ? 'HH:mm' : 'h A';
+    const defaultFinals = getDefaultFinalsStartDate();
+    const finalsDateFormat = defaultFinals ? 'ddd MM/DD' : 'ddd';
+    const date = showFinalsSchedule ? defaultFinals : new Date(2018, 0, 1);
```

Comment:
```suggestion
    const date = defaultFinals ?? new Date(2018, 0, 1);
```
If finals aren't being shown, the value of date doesn't matter, so this can be simplified. 

_The code above is also written assuming you choose to return `undefined` from `getDefaultFinalsStartDate()` like described below. The unneeded `showFinalsSchedule` is still true no matter what, unless I've missed something (very possible!)._